ver/us/splat.yaml

```diff
@@ -2161,7 +2161,8 @@ segments:
     overlay: True
     subsegments:
     - [0x320c50, c, lakilester]
-    - [0x3239b0, data]
+    - [0x3239b0, .data, lakilester]
```

Comment:
please use capitals for the address values (I know the one above isn't, I've been trying to migrate these over time)
And for the .rodata, you don't need to specify that and the last one here can just be
`- [0x323A40]` and the .rodata will be implicit

Reply:
Well, you will be happy to see what I did then 💋